apollo-runtime/src/main/java/com/apollographql/android/cache/normalized/sql/SqlStore.java

```diff
@@ -0,0 +1,92 @@
+package com.apollographql.android.cache.normalized.sql;
+
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.database.sqlite.SQLiteStatement;
+
+import com.apollographql.android.cache.normalized.CacheStore;
+import com.apollographql.android.cache.normalized.Record;
+
+import java.sql.SQLException;
+
+import javax.annotation.Nullable;
+
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_KEY;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.COLUMN_RECORD;
+import static com.apollographql.android.cache.normalized.sql.ApolloSqlHelper.TABLE_RECORDS;
+
+final class SqlStore extends CacheStore {
+
+  // Database fields
+  SQLiteDatabase database; //exposed for testing Mike will fix if a better path can be found.
+  private final ApolloSqlHelper dbHelper;
+  private final String[] allColumns = {ApolloSqlHelper.COLUMN_ID,
+      ApolloSqlHelper.COLUMN_KEY,
+      ApolloSqlHelper.COLUMN_RECORD};
+  private final SQLiteStatement sqLiteStatement;
+  private final String insertStatement;
```

Comment:
I don't think I can since it depends on a compiled statement that is dependent on a database instance